xml/selinux.xml

```diff
@@ -406,11 +373,19 @@ SELinux-Legacy</command></screen>
   <screen>&prompt.sudo;<command>restorecon -R /</command></screen>
 
   <para>
-   Now you can put &selnx; into enforcing mode by setting the following parameter
-   in the &grub; configuration file: <parameter>enforcing=1</parameter>. For context,
-   see <xref linkend="ex-sec-selinux-grubcfg"/>.
-   Then edit <filename>/etc/selinux/config</filename> and make sure
-   <option>SELINUX=enforcing</option> is set.
+   Now you can put &selnx; into enforcing mode. For this, edit
+   <filename>/etc/selinux/config</filename> and set
+   <option>SELINUX=enforcing</option>.
+   As next step, edit <filename>/etc/default/grub</filename> and
+   add <literal>enforcing=1</literal> to <literal>GRUB_CMDLINE_LINUX_DEFAULT=</literal>.
+   The line should now contain the following parameters which are relevant for &selnx;:
+  </para>
+  <screen>security=selinux selinux=1 enforcing=1</screen>
+  <para>
+   Rebuild your &grub; configuration with
+   <command>grub2-mkconfig -o /boot/grub2/grub.cfg</command>.
+   At the next reboot, the &selnx; policy will be enforced
+   and access will be denied based on policy rules.
```

Comment:
Just editing /etc/selinux/config is enough. Please remove the part about modifying the grub config